src/main/rx/expresslrs_common.h

```diff
@@ -122,14 +130,71 @@ typedef struct elrsModSettings_s {
 typedef struct elrsRfPerfParams_s {
     int8_t index;
     elrsRfRate_e enumRate;        // Max value of 16 since only 4 bits have been assigned in the sync package.
-    int32_t sensitivity;            // expected RF sensitivity based on
-    uint32_t toa;                   // time on air in microseconds
-    uint32_t disconnectTimeoutMs;   // Time without a packet before receiver goes to disconnected (ms)
-    uint32_t rxLockTimeoutMs;       // Max time to go from tentative -> connected state on receiver (ms)
-    uint32_t syncPktIntervalDisconnected; // how often to send the SYNC_PACKET packet (ms) when there is no response from RX
-    uint32_t syncPktIntervalConnected;    // how often to send the SYNC_PACKET packet (ms) when there we have a connection
+    int16_t sensitivity;            // expected RF sensitivity based on
+    uint16_t toa;                   // time on air in microseconds
+    uint16_t disconnectTimeoutMs;   // Time without a packet before receiver goes to disconnected (ms)
+    uint16_t rxLockTimeoutMs;       // Max time to go from tentative -> connected state on receiver (ms)
+    uint16_t syncPktIntervalDisconnected; // how often to send the SYNC_PACKET packet (ms) when there is no response from RX
+    uint16_t syncPktIntervalConnected;    // how often to send the SYNC_PACKET packet (ms) when there we have a connection
 } elrsRfPerfParams_t;
 
+typedef struct elrsFhssConfig_s {
+    uint8_t  domain;
+    uint32_t freqStart;
+    uint32_t freqStop;
+    uint8_t freqCount;
+} elrsFhssConfig_t;
+
+typedef struct elrsOtaPacket_s {
+    // The packet type must always be the low two bits of the first byte of the
+    // packet to match the same placement in OTA_Packet8_s
+    uint8_t type : 2,
```

Comment:
Our [coding style guide](https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md) doesn't mention bitfields, but the only place we use them is for ELRS support. Bitfields are great from a convenience perspective, but leave us open to endian issues and aren't generally considered good for portability. I'd be interested in input from other devs on if we should be using them. I'm included to suggest we consider eliminating them in another PR subsequent to this one. Thoughts?

Reply:
I'd take your advice on this, Steve.
Perhaps @ledvinap and @daleckystepan may be more helpful than I can be.

Reply:
@SteveCEvans there are more occurrences in the codebase:
https://github.com/betaflight/betaflight/blob/master/src/main/rx/cc2500_frsky_x.c#L73
https://github.com/betaflight/betaflight/blob/master/src/main/rx/crsf.c#L110
https://github.com/betaflight/betaflight/blob/master/src/main/rx/ghst_protocol.h#L122

Might be worth to fix them up all at once in a separate PR.

Reply:
As suggested, let's come back to this.